### DISCUSSION TITLE
AIO - Install - Behind NPM - No Connectivity w. Nextcloud Talk

### DISCUSSION THREAD

**Discussion by geoff-scott** [nextcloud/all-in-one#2110] [Questions] [answered]
Hello everyone, perhaps you guys can help me.

I have a nextcloud aio installation behind a NPM  (NGINX Proxy Manager) install pointing at port 11000 (as per documentation). Everything works respectively well except nextcloud talk.  It has never has been able to connect two people together in a call ever since day one of install. I would like to know why, and how to fix it...as I am assuming that this is a heavily used feature that should work out of the box...since it was shipped with it.

Text chat works fine, just not video or audio.

Some info about system setup.

VPS - Contabo
Spec: 4 Core - 8GB RAM
800GB SSD
Nextcloud AIO - 4.5.0
Nextcloud Install 25.0.4
UFW enabled with 3478, 80, 443, 11000 allowed through.

I am a novice when it comes to configuring nextcloud aio or anything inside a docker system.

I will need some serious step by step instructions if you ask me to do something for diagnostics etc.

**Comment by Zoey2936** [comment]
can you please share the screenshots of your npm configuration?

**Reply by geoff-scott** [reply]
How would I do that?

Like I mentioned in my post, I would need to know how to display the configuration, in order to screenshot for you.

**Reply by Zoey2936** [reply]
Please open your npm, open the proxy hosts, click edit on your cloud host and make screenshots of all 4 tabs

**Reply by geoff-scott** [reply]
Like this?

![image](https://user-images.githubusercontent.com/71340280/223023834-da476749-0f83-446c-b3bd-ab9bc1c392c6.png)
![image](https://user-images.githubusercontent.com/71340280/223023889-3e6abb88-e91c-4473-8f33-39f9bd75d765.png)
![image](https://user-images.githubusercontent.com/71340280/223023925-970c5605-8010-471b-9901-f40a6378e2b5.png)
![image](https://user-images.githubusercontent.com/71340280/223023949-eaf1615d-f5ce-4c30-bc30-f0454de644e4.png)

**Reply by Zoey2936** [reply]
Is your reverse proxy running on the same server as the nextcloud?

**Reply by geoff-scott** [reply]
No, different computer and different ip

**Reply by geoff-scott** [reply]
https://github.com/nextcloud/all-in-one/discussions/1986

Removing the High Speed Backend did the trick.

**Comment by szaimen** [comment]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Comment by szaimen** [comment, marked as answer]
I suppose this is the reason why it doesnt work: https://github.com/nextcloud/talk-android/issues/2704
